src/custom-elements/src/components/search-input/search-input.tsx

```diff
@@ -165,6 +165,14 @@ export class SearchInput {
     const newValue = (valueChange.target as HTMLInputElement).value;
     const isAutocomplete = this._isAutocomplete();
 
+    if (!newValue) {
+      this.selectedItem = undefined;
+
+      const dropdown = this._getAutocompleteDropdown();
+
+      dropdown.resetSelection();
```

Comment:
For reusability, we can extract this to a function (e.g. `resetSelection`) and use it also in lines 272-280

Reply:
Good one, done